src/Components/SmartComponents/BusinessRiskModal/BusinessRiskModal.js

```diff
@@ -0,0 +1,117 @@
+import { Button, Form, FormGroup, Modal, Radio, Stack, StackItem, TextInput } from '@patternfly/react-core';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications';
+import propTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { setBusinessRisk } from '../../../Helpers/APIHelper';
+import { businessRiskOptions } from '../../../Helpers/constants';
+
+const BusinessRiskModal = ({ cves, updateRef, ...props }) => {
+    const [cveList, setCveList] = useState(cves);
+    const [businessRiskId, setBusinessRiskId] = useState('0');
+    const [label, setLabel] = useState();
+
+    useEffect(getDefaultBusinessRisk, [cves]);
+
+    const createNotification = () => {
+        return props.addNotification({
+            variant: 'success',
+            title: `Business risk updated for ${cveList.length} CVEs`
+        });
+    };
+
+    const handleOnRadioChange = (_, event) => {
+        const { value } = event.currentTarget;
+        setBusinessRiskId(value);
+    };
+
+    const handleOnLabelChange = value => {
+        setLabel(value);
+    };
+
+    const handleClose = () => {
+        setCveList(undefined);
+    };
+
+    const handleSave = () => {
+        setBusinessRisk({
+            business_risk_id: parseInt(businessRiskId),
+            cve: cveList.map(item => item.id),
+            business_risk_text: label
+        })
+        .then(updateRef)
+        .then(() => createNotification());
+
+        handleClose();
+    };
+
+    function getDefaultBusinessRisk() {
+        const isSame = cveList.every(item => item.business_risk_id === cveList[0].business_risk_id);
```

Comment:
@jiridostal I'm not sure I understood why it's needed to check if the business_risk it's the same. Also,  it seems that it will always return true since in the `cveList`  all CVEs have `business_risk_id = 0 `

Reply:
This is so correct. fixed.